src/routes/Dataset/locales/fr.json

```diff
@@ -32,7 +32,9 @@
       "publisher": "Diffuseur",
       "author": "Auteur",
       "notDefined": "Inconnu",
-      "other": "Autre"
+      "other": "Autre",
+      "principalInvestigator": "Chercheur principal",
```

Comment:
Mauvaise traduction je pense

Reply:
@jdesboeufs ?

Reply:
la description en anglais c’est “key party responsible for gathering information and conducting research”